seed/lib/geospatial/eeej.py

```diff
@@ -30,82 +31,51 @@ def import_hud():
     # Note: instead of update_or_create, it might be better to clear and start over
     # Use smaller files to test with
     if settings.EEEJ_LOAD_SMALL_TEST_DATASET:
-        HUD_DATA_PATH_HOUSING = os.path.join(settings.BASE_DIR, 'seed/lib/geospatial/data', 'test-Public_Housing_Developments.xlsx')
-        HUD_DATA_PATH_MULTIFAMILY = os.path.join(settings.BASE_DIR, 'seed/lib/geospatial/data', 'test-Multifamily_Properties_-_Assisted.xlsx')
+        HUD_DATA_PATH_HOUSING = os.path.join(settings.BASE_DIR, 'seed/lib/geospatial/data', 'test-Public_Housing_Developments.csv.xz')
```

Comment:
I miss the ability to just look at these files, not sure this optimization is that big a deal when we account for the git compression that happens.

Reply:
On Windows with 7-Zip at least it's as easy as double-clicking, not sure what the mac experience is like